src/actions/genSale/purchaseDomains.ts

```diff
@@ -0,0 +1,121 @@
+import * as ethers from "ethers";
+import { getSaleStatus } from ".";
+import { GenSale } from "../../contracts/types";
+import { Claim, SaleStatus, Mintlist, Maybe, GenSaleStatus } from "../../types";
+
+const abi = ["function masterCopy() external view returns (address)"];
+
+const errorCheck = (condition: boolean, errorMessage: string) => {
+    if (condition) {
+        throw new Error(errorMessage);
+    }
+};
+
+const generateAccessList = (
+    userAddress: string,
+    gnosisSafeProxyAddress: string,
+    gnosisSafeImplAddress: string
+) => {
+    return [
+        {
```

Comment:
Not sure why this older code is formatted differently but it's unused anyways. Either delete it or leave a comment explaining why we've left it even though it's unused (i.e. there is a metamask bug when performing txs that require access lists)

Reply:
Ill remove it, you're correct; Its no longer used.